boot/zephyr/include/flash_map_backend/flash_map_backend.h

```diff
@@ -8,7 +8,7 @@
 #ifndef __FLASH_MAP_BACKEND_H__
 #define __FLASH_MAP_BACKEND_H__
 
-#include <flash_map.h> // the zephyr flash_map
+#include <storage/flash_map.h> // the zephyr flash_map
```

Comment:
Moving `sim/mcuboot-sys/csupport/flash_map.h` -> `sim/mcuboot-sys/csupport/storage/flash_map.h` should probably fix this breakage. Otherwise you can use `defined(__BOOTSIM__)` here...

Reply:
I moved the file as prefer not to increase if-def-ry